meta.go

```diff
@@ -335,15 +412,12 @@ func setMetaValueRecursive(key string, value string, previousMeta interface{}, j
 
 // setupDir makes directory and json file for meta
 func setupDir(metaSpace string, metaFile string) error {
```

Comment:
`staticcheck` [1] says: `meta.go:414:6: func setupDir is unused (U1000)`.  I think you have `MetaSpec.SetupDir` now - can this be deleted?

```
$ go doc github.com/screwdriver-cd/meta-cli.MetaSpec.SetupDir
func (m *MetaSpec) SetupDir() ([]byte, error)
    SetupDir creates the metaspace directory and writes a file with empty
    object.
```

[1] http://staticcheck.io/docs/

Reply:
Yep; done